src/plugins/hetero/src/compiled_model.cpp

```diff
@@ -0,0 +1,719 @@
+            xmlString.resize(dataSize);
+            model.read(const_cast<char*>(xmlString.c_str()), dataSize);
+
+            /// read blob content
+            ov::Tensor weights;
+            model.read(reinterpret_cast<char*>(&dataSize), sizeof(dataSize));
+            if (0 != dataSize) {
+                weights = ov::Tensor(ov::element::from<char>(), ov::Shape{static_cast<ov::Shape::size_type>(dataSize)});
+                model.read(weights.data<char>(), dataSize);
+            }
+
+            ov_model = plugin->get_core()->read_model(xmlString, weights);
+            compiled_model = plugin->get_core()->compile_model(ov_model, device, loadConfig);
+        }
+
+        m_compiled_submodels.emplace_back(ov::hetero::CompiledModel::CompiledModelDesc{
+            device,
+            ov_model,
+            compiled_model,
+        });
+    }
+
+    auto inputs_map_node = heteroNode.child("inputs_to_submodels_inputs");
+    FOREACH_CHILD (xml_node, inputs_map_node, "pair") {
+        m_inputs_to_submodels_inputs.emplace_back(GetUInt64Attr(xml_node, "submodel_idx"),
+                                                  GetUInt64Attr(xml_node, "node_idx"));
+    }
+    auto outputs_map_node = heteroNode.child("outputs_to_submodels_outputs");
+    FOREACH_CHILD (xml_node, outputs_map_node, "pair") {
+        m_outputs_to_submodels_outputs.emplace_back(GetUInt64Attr(xml_node, "submodel_idx"),
+                                                    GetUInt64Attr(xml_node, "node_idx"));
+    }
+    auto submodels_input_to_prev_output_node = heteroNode.child("submodels_input_to_prev_output");
+    FOREACH_CHILD (xml_node, submodels_input_to_prev_output_node, "record") {
+        std::pair<uint64_t, uint64_t> in_pair = {GetUInt64Attr(xml_node, "in_submodel_idx"),
+                                                 GetUInt64Attr(xml_node, "in_node_idx")};
+        std::pair<uint64_t, uint64_t> out_pair = {GetUInt64Attr(xml_node, "out_submodel_idx"),
+                                                  GetUInt64Attr(xml_node, "out_node_idx")};
+        m_submodels_input_to_prev_output.emplace(in_pair, out_pair);
+    }
+    set_inputs_and_outputs();
+}
+
+std::shared_ptr<ov::ISyncInferRequest> ov::hetero::CompiledModel::create_sync_infer_request() const {
+    return std::make_shared<ov::hetero::InferRequest>(
+        std::static_pointer_cast<const ov::hetero::CompiledModel>(shared_from_this()));
+}
+
+std::shared_ptr<ov::IAsyncInferRequest> ov::hetero::CompiledModel::create_infer_request() const {
+    auto internal_request = create_sync_infer_request();
+    auto async_infer_request = std::make_shared<ov::hetero::AsyncInferRequest>(
+        std::static_pointer_cast<ov::hetero::InferRequest>(internal_request),
+        get_task_executor(),
+        get_callback_executor());
+
+    return async_infer_request;
+}
+
+void ov::hetero::CompiledModel::set_property(const ov::AnyMap& properties) {
+    m_cfg = Configuration{properties, m_cfg};
```

Comment:
HETERO 1.0 does not support set properties (former `SetConfig`).
Do we have now properties which can be changed in runtime?

Reply:
As far as I saw, there was an overridden `SetConfig` - https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/hetero/plugin.cpp#L175

Reply:
It's for plugin, not for executable network.

Reply:
You are right, changed implementation with `OPENVINO_NOT_IMPLEMENTED`